src/main/steps/applicant1/hub-page/joint/content.ts

```diff
@@ -289,24 +299,45 @@ export const generateContent: TranslationFn = content => {
     ? userCase.applicant2AppliedForFinalOrderFirst === YesOrNo.YES
     : userCase.applicant1AppliedForFinalOrderFirst === YesOrNo.YES;
 
-  const displayState = currentStateFn(userCase.state).at(
-    (userCase.state === State.OfflineDocumentReceived ? userCase.previousState : userCase.state) as State
-  );
+  const displayStateAsState = (
```

Comment:
you can just call it caseState or something. the name displayState was created mainly for the sequence state